src/app/parcels/ActionBar/ActionsModal.tsx

```diff
@@ -182,11 +198,24 @@ const ActionsModal: React.FC<ActionsModalProps> = (props) => {
                 {props.errorText && <small>{props.errorText}</small>}
                 {loadPdf ? (
                     <>
-                        <Heading> The PDF is ready to be downloaded. </Heading>
+                        {props.downloadable ? (
```

Comment:
instead of having a boolean `downloadable`, should we have a enum of some sort like
```
type ActionType = 'pdfDownload' | 'deleteParcel'
```
to be clearer?

Reply:
Yes that's much better, thanks